CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+# v0.1.1
+
+* Add support for Redox/Orbital.
+* Add support for BSD distributions.
+* Ported Windows backend from `winapi` to `windows-sys`.
+* **Breaking:** Take a reference to a window instead of owning the window.
+* Add a `from_raw` function for directly using raw handles.
+* **Breaking:** Add feature flags for `x11` and `wayland` backends.
+* Use static dispatch instead of dynamic dispatch for the backends.
+* Add `libxcb` support to the X11 backend.
```

Comment:
Copy the lines from README.md for 0.1.1 and 0.1.0, then link to this file from the README.md